api/src/main/java/io/opentelemetry/trace/DefaultTracerFactory.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.trace;
+
+public class DefaultTracerFactory implements TracerFactory {
+
+  private static final TracerFactory instance = new DefaultTracerFactory();
+
+  public static TracerFactory getInstance() {
+    return instance;
+  }
+
+  @Override
+  public Tracer get(String instrumentationName) {
+    return DefaultTracer.getInstance();
```

Comment:
Nit: Usually you have the concrete implementation on the most generic function and the others call in that instead of this mode.

Reply:
agreed.  fixed.